wordpress-mobile/WordPress-Android
# Automated Transfers v1

This PR adds the ability to install plugins for wp.com (non-atomic) business plan sites. Trying to install a plugin will start an Automated Transfer. Its steps are explained as comments in `PluginDetailActivity`. The changes to FluxC are added in https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/775. Although it'd be good to check out the description of that PR, it's not necessary to review this one or to merge it since it's targeting a WIP branch.

To test:
1. Create a new business plan site on WordPress.com
2. Go into the plugins page for that site
3. Tap on any plugin and try to install it
4. Verify that you get an error
5. Add a custom domain to your site
6. Try to install the plugin again, make sure the progress bar is updated and the view refreshes after it's complete (the install button should say `Installed` for example)
7. Go back to the plugin directory page and make sure the site plugins showed up
